spec/support/schemas/claims_api/v2/errors/default.json

```diff
@@ -19,6 +19,10 @@
             "type": "string",
             "description": "HTTP error status code"
           },
+          "code": {
```

Comment:
Added this because the common 502 exception includes both code and status despite being the same thing and I didn't want to mess. If you think this could cause issues for the front end or if there's API docs I need to update let me know